EasyPost.Tests/ShipmentTest.cs

```diff
@@ -73,42 +44,145 @@ public async Task TestBuy()
             _vcr.SetUpTest("buy");
 
 
-            Shipment shipment = await Shipment.Create(Fixture.FullShipment);
+            Shipment shipment = await CreateFullShipment();
 
             await shipment.Buy(shipment.LowestRate());
 
             Assert.IsNotNull(shipment.postage_label);
         }
 
         [TestMethod]
-        public async Task TestRegenerateRates()
+        public async Task TestConvertLabel()
         {
-            _vcr.SetUpTest("regenerate_rates");
+            _vcr.SetUpTest("convert_label");
 
+            Shipment shipment = await CreateOneCallBuyShipment();
 
-            Shipment shipment = await Shipment.Create(Fixture.FullShipment);
+            await shipment.GenerateLabel("ZPL");
 
-            await shipment.RegenerateRates();
+            Assert.IsNotNull(shipment.postage_label.label_zpl_url);
+        }
 
-            List<Rate> rates = shipment.rates;
+        [TestMethod]
+        public async Task TestCreate()
```

Comment:
Can we keep the order here too? Moving functions around isn't ideal because it blows up the lines to review and harder to determine what actually changed vs what got moved.

Reply:
I have reverted the rearrangement manually.

One of the pre-commit checks on my IDE is to "rearrange code", which helps keep code in line with our style guide, in terms of in what order they appear. This was recently enabled on my end so I didn't have to remember to do this manually. Clearly, already our code base has strayed from the style guide.

Currently, per our style guide, code is arranged by access level, by static/non-static, and alphabetically.

After this PR is merged, I'm going to run a system-wide lint of this code base and open a separate PR. There will be no code or functionality changes in it, just rearrangement to fall in line with our style guide. And with the pre-commit check, moving forward, things should never need to be re-ordered again.